bin/anthology/papers.py

```diff
@@ -156,19 +157,11 @@ def volume_id(self):
 
     @property
     def paper_id(self):
-        if self.collection_id[0] == "W" or self.collection_id == "C69":
```

Comment:
This changes the semantics of paper_id if I understand correctly. Could that change anything in non-obvious places?

Maybe this would be a good point to make naming consistent: `paper_ids` in volume return full ids and should therefore probably be renamed to `full_paper_ids`.

This is not blocking the merge, just want to note it before it is forgotten again.

Reply:
I looked at all calls to paper ID and updated them.

IMO, `paper_id` should return the paper number, analogous to `anthology_id` and `collection_id`. Then, `full_id` should be renamed `anthology_id`.

Reply:
I think this is resolved.